<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>393</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="codeEditor" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(O)</string>
    </property>
    <addaction name="actionAutoWordWrap"/>
    <addaction name="actionShowLine"/>
    <addaction name="actionFont"/>
    <addaction name="actionFontColor"/>
    <addaction name="separator"/>
    <addaction name="actionFontBackColor"/>
    <addaction name="actionBackColor"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(V)</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionShowBar"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSaveFile"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionShowLine"/>
   <addaction name="separator"/>
   <addaction name="actionToolBar"/>
   <addaction name="actionShowBar"/>
  </widget>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/note_add.png</normaloff>:/icons/iconsFile/note_add.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/folder_open.png</normaloff>:/icons/iconsFile/folder_open.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/save.png</normaloff>:/icons/iconsFile/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/save_as.png</normaloff>:/icons/iconsFile/save_as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/redo.png</normaloff>:/icons/iconsFile/redo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/undo.png</normaloff>:/icons/iconsFile/undo.png</iconset>
   </property>
   <property name="text">
    <string>恢复()</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/content_cut.png</normaloff>:/icons/iconsFile/content_cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/content_copy.png</normaloff>:/icons/iconsFile/content_copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/content_paste.png</normaloff>:/icons/iconsFile/content_paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/search.png</normaloff>:/icons/iconsFile/search.png</iconset>
   </property>
   <property name="text">
    <string>查找(F)</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/replace.png</normaloff>:/icons/iconsFile/replace.png</iconset>
   </property>
   <property name="text">
    <string>替换(R)</string>
   </property>
   <property name="toolTip">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选(A)</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAutoWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/auto_wrap.png</normaloff>:/icons/iconsFile/auto_wrap.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(W)</string>
   </property>
   <property name="toolTip">
    <string>自动换行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShowLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/chapter.png</normaloff>:/icons/iconsFile/chapter.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(L)</string>
   </property>
   <property name="toolTip">
    <string>显示行号</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/font.png</normaloff>:/icons/iconsFile/font.png</iconset>
   </property>
   <property name="text">
    <string>字体(F)</string>
   </property>
   <property name="toolTip">
    <string>字体</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="toolTip">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="actionFontBackColor">
   <property name="text">
    <string>字体背景色</string>
   </property>
   <property name="toolTip">
    <string>字体背景色</string>
   </property>
  </action>
  <action name="actionBackColor">
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="toolTip">
    <string>编辑器背景色</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/Toolbar.png</normaloff>:/icons/iconsFile/Toolbar.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(T)</string>
   </property>
   <property name="toolTip">
    <string>工具栏</string>
   </property>
  </action>
  <action name="actionShowBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconsFile/zhuangtailan.png</normaloff>:/icons/iconsFile/zhuangtailan.png</iconset>
   </property>
   <property name="text">
    <string>状态栏(S)</string>
   </property>
   <property name="toolTip">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>codeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
